ui/app/pages/permissions-connect/choose-account/index.scss

```diff
@@ -9,26 +9,76 @@
     @extend %header--18;
   }
 
-  &__text {
+  &__text, &__text--blue, &__text--grey {
     @extend %content-text;
     line-height: 25px;
   }
 
+  &__text-blue {
+    color: $curious-blue;
+    cursor: pointer;
+  }
+
+  &__text-grey {
+    color: $Grey-500;
+  }
+
   &__accounts-list {
-    width: 393px;
+    width: 428px;
     border: 1px solid #D0D5DA;
     box-sizing: border-box;
     border-radius: 8px;
-    margin-top: 36px;
+    margin-top: 8px;
+    max-height: 338px;
+    overflow-y: scroll;
 
     @media screen and (max-width: 575px) {
       width: 100%;
     }
   }
 
+  &__accounts-list-header {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 36px;
+    width: 426px;
```

Comment:
Nit: Instead of setting the width to `426px` separately for each of these pieces, could this instead be set on the container, and each of these be set to 100%?

Either that or using a variable. But setting it on the container seems like it'd make more sense.

Reply:
fixed in 532b5bf4d